includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -31,8 +31,8 @@ class EDD_Customer_Addresses_Table extends List_Table {
 	 */
 	public function __construct() {
 		parent::__construct( array(
-			'singular' => __( 'Address',   'easy-digital-downloads' ),
-			'plural'   => __( 'Addresses', 'easy-digital-downloads' ),
+			'singular' => 'address',
```

Comment:
Are these intentionally no longer translatable strings (same for emails)? I thought these were label definitions.